CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
I am not entirely sure if it is necessary, but we could probably increase this quite a bit more.

Reply:
I don't mind, 3.5 is the minimum version to silence the warning

Reply:
EDM4hep already requires 3.16 so I think it is safe to go to that as well.